sql/create.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TABLE product
+(
+  id SERIAL PRIMARY KEY,
+  name Varchar(50) NOT NULL,
+  price MONEY NOT NULL,
+  vat NUMERIC(2,2) NOT NULL,
+  quantity INTEGER NOT NULL CHECK (quantity > 0),
+  CHECK (vat < 1.0),
+  CHECK (vat >= 0)
+);
+
+CREATE TABLE basket
+(
+  id SERIAL PRIMARY KEY
+);
```

Comment:
@gotar Because I wanted to create a database first, fill it with data and then try to connect it with my current ruby code. Should I make another branch for it?